### PR TITLE
Implants that grant spells to their owners, for adminbus

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
:cl: Naksu
admin: Admins can now grant spells via implants, using the spell implant. Some VV is required.
/:cl:

### CODE REVIEW

File: code/game/objects/items/implants/implant_spell.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		target.RemoveSpell(spell)
+		if(target.stat != DEAD && !silent)
+			to_chat(target, "<span class='boldnotice'>The knowledge of how to cast [spell] slips out from your mind.</span>")
+
+/obj/item/implanter/spell
+	name = "implanter (spell)"
+	imp_type = /obj/item/implant/spell
+
+/obj/item/implantcase/spell
+	name = "implant case - 'Krav Maga'"
```
Comment (XDTM) [inline code/game/objects/items/implants/implant_spell.dm]: Martial wizardry

Comment (vuonojenmustaturska) [reply code/game/objects/items/implants/implant_spell.dm]: copy and paste strikes again

**Comment by Dorsisdwarf** [comment]
if this is only an admin tool, what's the point? admins can already give people spells.

**Comment by XDTM** [comment]
Maybe as a physical object? That way you can put it in a maze and the first one who gets there gets the spell, or you can give it to traders to sell

**Comment by ExcessiveUseOfCobblestone** [comment]
we already have oneuse books

**Comment by vuonojenmustaturska** [comment]
these aren't oneuse, they're removable (and transferable)

**Comment by KingPhilipIII** [comment]
PRetty sure books can be recharged too.

**Comment by AlexTheSysop42** [comment]
There's an 80% chance to burn the book

**Comment by Anonmare** [comment]
Implants are bound to the body while books bind the spell to the mind

if you die, you have to either extract the implant or get a new one

**Comment by Shadowflame909** [comment]
do these implants require wizard clothing? and would this make it more likely for admins to accept za tc trade. Syndicate spell implant for 20 tc.

**Comment by vuonojenmustaturska** [comment]
There's a var to automatically make the spell robeless on the implant, it defaults to true. Whether admins are more or less likely to do what you want is not influenced by this PR.

Basically the PR is more about completion, so you can do low-key events like experimented-on corpses washing up on the station or whatever. To do the same thing with books you'd have to keep tracking who has the spell manually and when they croak transfer the spell to someone else, or recharge the book.

**Comment by ExcessiveUseOfCobblestone** [comment]
Is what you're trying to achieve spell transfer via surgery, because it's a lot easier to go through 2 clicks (add/remove, pick the spell) than it is to manipulate organs/implants.

**Comment by AlexTheSysop42** [comment]
books can only do certain spells though

**Comment by vuonojenmustaturska** [comment]
@ExcessiveUseOfCobblestone it's more about player agency really, if you plop some of these bad boys on the station you don't really know what's going to happen and who's going to end up with the implants. The players can self-serve when they want implants transferred and they don't have to break the 4th wall to ask an admin to do it for them.

**Pull request merged**